roles/tackle/defaults/main.yml

```diff
@@ -178,6 +176,7 @@ maven_data_volume_claim_name: "{{ app_name }}-{{ maven_name }}-volume-claim"
 # RH-SSO specific
 rhsso_name: "rhsso"
 rhsso_service_name: "{{ app_name }}-{{ rhsso_name }}"
+rhsso_secret_name: "credential-{{ app_name }}-{{ rhsso_name }}"
```

Comment:
We could re-use "credential-{{ rhsso_service_name }} which is esentially how the secret will be derived on this one